models/migrations/v80.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"os"
+
+	"code.gitea.io/gitea/models"
+	"code.gitea.io/gitea/modules/setting"
+	"github.com/go-xorm/xorm"
+)
+
+func deleteOrphanedAttachments(x *xorm.Engine) error {
```

Comment:
I don't think migration is correct place for this, it should be periodical job like gitfsck, ldap sync etc

Reply:
This migration is to delete all orphaned attachments from previous releases, not something that should be run periodically.

I get your point though, a periodic job to clear future orphaned attachments would make sense

Reply:
But in such case it won't delete orphaned attachments (those who was added to releases that are already deleted). You would have to use where condition something like:
```
`comment_id` = 0 and `release_id` > 0 and `release_id` not in (select `id` from `releases`)
```